test/e2e/persistent_volumes.go

```diff
@@ -662,3 +812,94 @@ func makeWritePod(ns string, pvcName string) *api.Pod {
 		},
 	}
 }
+
+// Generate a list of PV specs.
+// pvsToCreate should be the number of PVs desired
+// Returns a list of generated PV specs.  This can be passed to createMultiplePVs
+func makeMultiplePersistentVolumes(serverIP string, pvsToCreates int) []*api.PersistentVolume {
```

Comment:
This func does not mirror makePersistentVolume()'s ability to include a ClaimRef in the pv. Since the claimRef logic is a bit convoluted  (two funcs deal with it) maybe it's better to yank the pvc arg from makePersistentVolume() and put all of the pre-bound claimRef logic in createPVCPV() ?